openapi.yml

```diff
@@ -1389,6 +1387,11 @@ components:
         width:
           description: Width in pixels
           type: integer
+    Purl:
+      description: Stanford persistent URL associated with the related resource.
+      type: string
+      format: uri
+      pattern: '^http://'
```

Comment:
Would you mind dropping in a comment here that says "Arcadia has stated that the canonical URL will be http, even though this redirects to https"

Reply:
Added.